client/java-client-osgi/shared.bnd

```diff
@@ -5,3 +5,4 @@ Bundle-Version: ${parsedVersion.osgiVersion}
 # BND configuration to tweak generation of Manifest entries for the normal / sources JAR archive
 -removeheaders: Bnd-LastModified,Bundle-Developers,Bundle-DocURL,Bundle-SCM,Include-Resource,Private-Package
 -noimportjava: true
+-fixupmessages "Classes found in the wrong directory"; restrict:=error; is:=warning
```

Comment:
Why is this line needed in this context? Could we add a comment maybe?

Reply:
Added in [2a64f42](https://github.com/SonarSource/sonarlint-core/pull/973/commits/2a64f423d36f5f55e1ca9c31ae6e8f15ecb476f2)